fscryptctl.c

```diff
@@ -61,6 +61,10 @@ static void secure_wipe(void *v, size_t n) {
 #define FSCRYPT_KEY_DESCRIPTOR_HEX_SIZE ((2 * FSCRYPT_KEY_DESCRIPTOR_SIZE) + 1)
 #define FSCRYPT_KEY_IDENTIFIER_HEX_SIZE ((2 * FSCRYPT_KEY_IDENTIFIER_SIZE) + 1)
 
+#ifndef VERSION
```

Comment:
I think this should go closer to the top of the file, so it isn't mixed in among other things.

Reply:
Done